src/Components/ReviewsList.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import ReviewListing from './ReviewListing'; 
+import { fetchReviewList } from '../Utils/api';
+
+const ReviewList = () => {
+    const [reviews, setReviews] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+
+    useEffect(() => {
+        fetchReviewList()
+        .then((data) => {
+            setReviews(data);
+            setIsLoading(false);
+        });
+    }, []);
+
+
+    if (isLoading) return <p>Loading Reviews... please wait...</p>
+    return (
+        <section id="reviewList">
+            {reviews.reviews.map((review) => {
+                return (
+                    <ol type="1" key={review.review_id}>
+                        <ReviewListing
```

Comment:
Your \<ol> tag here is rendering \<ReviewListing> as a child of it, but ReviewListing returns an article tag which means the direct child of \<ol> would be an article tag. Ordered and Unordered lists should only ever have list items as their direct children. Feel free to wrap a \<li> tag around the article tag in review listing to keep it semantic, but think more carefully about this in future

Reply:
Ok I have changed the code, and removed the <ol> from around ReviewListing, and I have put the <li> around the article tag in the ReviewListing component.